core/controllers/reader.py

```diff
@@ -294,6 +298,69 @@ def get(self, exploration_id):
         self.render_json(self.values)
 
 
+class StorePlaythroughHandler(base.BaseHandler):
+    """Handles a useful playthrough coming in from the frontend to store it."""
+
+    @acl_decorators.can_play_exploration
```

Comment:
Why "play"? Shouldn't it at least be edit access?

Reply:
@seanlip Nope. A playthrough of a learner is what's being stored right? The learner user doesnt need to have edit access right?

Reply:
Why do you want to allow learners to look at other learners' playthroughs?

Reply:
@seanlip Im a bit confused here. What does this decorator mean? Doesnt it mean that the user that is sending the request should have that particular permission? So, the StorePlaythroughController is sent from the front end during a learner session and technically from the learner user. Why would he need edit access? 
Please correct me if Im understanding this wrong here.

Reply:
Oh, sorry! It's me who understood it wrongly. You're correct. Thank you for pushing back.